drivers/usb_c/tcpc/rt1715.c

```diff
@@ -427,8 +427,9 @@ static int rt1715_tcpc_mask_status_register(const struct device *dev, enum tcpc_
 static int rt1715_tcpc_set_drp_toggle(const struct device *dev, bool enable)
 {
 	const struct rt1715_cfg *cfg = dev->config;
+	uint8_t pd_int_rev = 0x10;
```

Comment:
You should read this revision from the chip register directly during the init function and save it in the `struct rt1715_data` of the driver.